### DISCUSSION TITLE
NEXTAUTH_URL

### DISCUSSION THREAD

**Discussion by nikitapilgrim** [nextauthjs/next-auth#2377] [Help]
### Question 💬

Hello, thank you for the library. 
I have a problem with NEXTAUTH_URL in production. In dev mode I just set localhost:3000.  
I specify it because I use api route proxy and Providers.Credentials

What should I set for production vercel?

### How to reproduce ☕️

process.env.NEXTAUTH_URL = 'http://localhost:3000';

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
Depends. If you host on Vercel, it should be OK, nothing needs to be set. If you use another hosting solution, it should be the URL of your website, like `https://example.com`
